src/server.ts

```diff
@@ -35,6 +33,9 @@ import { setupArchiverDiscovery } from '@shardus/archiver-discovery'
 import { setDefaultResultOrder } from 'dns'
 import { nestedCountersInstance } from './utils/nestedCounters'
 import { methodWhitelist } from './middlewares/methodWhitelist'
+import { isDebugModeMiddlewareLow, rateLimitedDebugAuth } from './middlewares/debugMiddleware'
+import { isIP, isIPv4 } from 'net'; 
```

Comment:
```suggestion
import { isIPv4 } from 'net'; 
```
remove unused import